test/controllers/test_executions_controller_test.rb

```diff
@@ -47,13 +57,13 @@ def setup_c4
     te.save!
 
     # add provider performance to each record associated with the measure test
-    provider = Provider.find('53b2c4414d4d32139c730000')
-    test.provider = provider
-    test.save!
-    test.records.each do |record|
-      pp = record.provider_performances.build(provider: provider)
-      pp.save!
-    end
+    # provider = Provider.find('53b2c4414d4d32139c730000')
```

Comment:
If these lines aren't necessary, should they be removed rather than commented out?